content/en/docs/Reference/reset.md

```diff
@@ -111,18 +111,6 @@ spec:
 
 It is possible to trigger the reset manually by logging into the recovery from the boot menu and running `kairos reset` from the console.
 
-To optionally change the behavior of the reset process (such as cleaning up also configurations), run `elemental reset` instead which supports options via arg:
-
-| Option              | Description                 |
-| ------------------- | --------------------------- |
-| --reset-persistent  | Clear persistent partitions |
-| --reset-oem         | Clear OEM partitions        |
-| --system.uri string | Reset with the given image  |
```

Comment:
Are these no longer possible? Do we need an issue to bring this functionality to `kairos-agent`?

Reply:
yep, was working on this on the side while testing all. Im now ondering if it would have been faster to bring elemental-cli fully wiht all cmds and merge agent into it directly lol

Reply:
created https://github.com/kairos-io/kairos/issues/1451